InstantPrintTool.py

```diff
@@ -16,7 +16,16 @@
 
 from ui.ui_printdialog import Ui_InstantPrintDialog
 
-
+class InstantPrintDialog(QDialog):
+    
+    hidden = pyqtSignal()
+    
+    def __init__(self):
+        QDialog.__init__(self)
```

Comment:
Change to 

    def __init__(self, parent):
        QDialog.__init__(self, parent)